debian/control

```diff
@@ -1,7 +1,8 @@
 Source: libndn-cxx
 Priority: optional
 Maintainer: Endless Maintainers <maintainers@endlessm.com>
-Build-Depends: debhelper (>= 9),
+Build-Depends: adduser,
```

Comment:
Why is adduser needed as a build dependency? AFAICT, the use of adduser is in ndnsec-tools.postinst, so this should be added to the ndnsec-tools `Depends`.